<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>weatherize 0.001</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="maxDateTime">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>700</y>
      <width>750</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>750</width>
      <height>530</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="minDateTime">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>670</y>
      <width>750</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartView2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>750</width>
      <height>530</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>630</y>
      <width>750</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Loading data..</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QRadioButton" name="ElectricityPro">
       <property name="text">
        <string>ElectricityPro</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="NuclearPro">
       <property name="text">
        <string>NuclearPro</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="Temp">
       <property name="text">
        <string>Temp</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="Wind">
       <property name="text">
        <string>Wind</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="Electricity">
       <property name="text">
        <string>Electricity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="WindFore">
       <property name="text">
        <string>WindFore</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="Hydro">
       <property name="text">
        <string>Hydro</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="TentativePro">
       <property name="text">
        <string>TentativePro</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="Cloud">
       <property name="text">
        <string>Cloud</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>680</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min. Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>710</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getData">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>740</y>
      <width>750</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Data</string>
    </property>
   </widget>
   <widget class="QChartView" name="chartView3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>780</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>average maximum temperature</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>average minimum temperature</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNumber3"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber2"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>average temperature</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SelectLocation">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>794</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="request">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>924</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Request Calculations</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
